roles/addons/prometheus_client/readme.rst

```diff
@@ -11,6 +11,33 @@ Instructions
 
 Node_exporter is available at http://localhost:9100
 
+exporters configuration file to add to inventory/group_vars/equipment_type_X
+depending on where the exporter should be:
```

Comment:
Does the role work out-of-the-box without these parameters?

Reply:
Without this, the role will do nothing.

Reply:
Can we consider adding this to defaults/main.yml?

Reply:
No, impossible for these ones. These need to be manually tuned for each kind of equipment, no defaults possible here.

Reply:
Ho ok got it, you mean an empty configuration. Will do it.